testsuite/libra-fuzzer/tests/coverage.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) The Libra Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use libra_fuzzer::FuzzTarget;
+use std::{env, fs, path::Path};
+
+#[test]
+fn coverage() {
+    // get path to corpus/ folder from env var CORPUS_PATH.
+    let corpus_path = env::var_os("CORPUS_PATH").expect("CORPUS_PATH env var should be provided");
```

Comment:
Can the location of the corpus dir be discovered through the fuzz target impls?

Reply:
not sure what you mean by that, I could also hardcode it as `Path::new(env::var("CARGO_MANIFEST_DIR")).join("/fuzz/corpus")`

initially I wasn't sure how the fuzzer peeps would use this and where they would store the new corpus, but I guess if they run our fuzzer, they probably will have the corpus there as well : )